rdmo/projects/assets/js/project/components/helper/Modal.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { Modal as BootstrapModal } from 'bootstrap'
+
+const Modal = ({
```

Comment:
I really like this Modal (I am looking for a good pattern for some while. We should move it to `core`. I improved it a bit and removed one ref:

```js
  const modalRef = useRef(null)

  useEffect(() => {
    const modalElement = modalRef.current
    if (!modalElement) return

    const modal = BootstrapModal.getOrCreateInstance(modalElement, {
      backdrop: 'static',
      keyboard: true,
      ...modalProps
    })

    if (show) {
      modal.show()
    }

    return () => modal.hide()
  }, [show])

  return (
    <div
      ref={modalRef}
```

the rest stays the same.